### PR TITLE
Correctly source header data

### PR DESCRIPTION
**Pull request by bradleybeddoes**
Correctly source `Authorization` header from `request.headers` instead of `request.env`

### CODE REVIEW

File: app/controllers/api/api_controller.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@
 require 'openssl'
 
 module API
-  class APIController < ActionController::Base
+  class APIController < ActionController::API
```
Comment (chrisjones120) [inline app/controllers/api/api_controller.rb:6]: What is happening here? I was under the impression that this should inherit `Base`, eg: https://github.com/ausaccessfed/aaf-gumboot/blob/develop/spec/dummy/app/controllers/api/api_controller.rb

Comment (bradleybeddoes) [reply app/controllers/api/api_controller.rb:6]: Nope, I had to take a copy of `APIController` into this project itself so I could do the Token work. That being the case the most appropriate basis is `ActionController::API` which, if we had a long life for Gumboot in mind I'd update it to as well.

Comment (bradleybeddoes) [reply app/controllers/api/api_controller.rb:6]: Well, the controller was here, I had to copy and re-work the specs, the voices in my head know what they mean....

Comment (chrisjones120) [reply app/controllers/api/api_controller.rb:6]: TIL about this, good to know.

**Pull request merged**